src/main/scala/cqrs/settings/Settings.scala

```diff
@@ -1,7 +1,7 @@
 package cqrs.settings
 
 import akka.actor.{ Actor, ExtensionKey, Extension, ExtendedActorSystem }
-import scala.concurrent.duration.{ Duration, FiniteDuration }
+import scala.concurrent.duration._
```

Comment:
I try to follow: http://twitter.github.io/effectivescala/#Formatting-Imports

*Use wildcards when more than six names are imported*